### PR TITLE
add doc for https://github.com/zalando-zmon/zmon-worker/pull/265

### PR DESCRIPTION
**Pull request by drummerwolli**
add doc for https://github.com/zalando-zmon/zmon-worker/pull/265

### CODE REVIEW

File: docs/user/notifications/hipchat.rst [hunk 1]
```diff
@@ -24,6 +24,12 @@ Notify Hipchat room with alert status.
     :param color: Message color. Default is ``red`` if alert is raised.
     :type color: str
 
+    :param link: Add link to Hipchat message. Default is ``False``.
+    :type color: bool
```
Comment (vetinari) [inline docs/user/notifications/hipchat.rst]: `type link`?

File: docs/user/notifications/hipchat.rst [hunk 2]
```diff
@@ -24,6 +24,12 @@ Notify Hipchat room with alert status.
...
+    
+    :param link_text: if ``link`` param is ``True``, this will be displayed as a link in the hipchat message. Default is  ``go to alert``.
+    :type color: str
```
Comment (vetinari) [inline docs/user/notifications/hipchat.rst]: `type link_text`?

File: docs/user/notifications/hipchat.rst (hunk 1 above)
Comment (drummerwolli) [reply docs/user/notifications/hipchat.rst]: ah, mixed up the syntax ... will fix

**Comment by vetinari** [comment]
:+1:

**Comment by femueller** [comment]
:+1:

**Pull request merged**